package.json

```diff
@@ -0,0 +1,45 @@
+{
+	"name": "metal-autocompleteBadges",
+	"version": "2.0.0",
+	"description": "An autocomplete component that renders badges for selected options",
+	"license": "BSD",
+	"repository": "DiogoMoreiraLins/metal-autocompleteBadges",
```

Comment:
Can you rename this to `metal/metal-autocomplete-badges`, since it will now be in this org? :D